cves/CVE-2010-2068.yml

```diff
@@ -190,11 +191,12 @@ mistakes:
     In your opinion, after all of this research, what mistakes were made that
     led to this vulnerability? Coding mistakes? Design mistakes?
     Maintainability? Requirements? Miscommunications?
-
     Look at the CWE entry for this vulnerability and examine the mitigations
     they have written there. Are they doing those? Does the fix look proper?
-
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    A lack of testing was the mistake for this vulnerability. Most of the code had tests except for the proxy subsystem. 
+    A lot of changes were made in between the fix and vcc without testing which was likely the reason that the vulnerability did not get caught sooner.
+    
```

Comment:
How did the vulnerability actually occur? Is there a lower level mistake that you could also include?

Reply:
Added some more information on the vulnerability and talked about a lower level mistake.